components/concurrency_manager/src/lib.rs

```diff
@@ -135,9 +135,9 @@ mod tests {
     #[tokio::test]
     async fn test_lock_keys_order() {
         let concurrency_manager = ConcurrencyManager::new(1.into());
-        let keys: Vec<_> = [b"c", b"a", b"b"]
-            .iter()
-            .map(|k| Key::from_raw(*k))
+        let keys: Vec<_> = vec![b"c", b"a", b"b"]
+            .into_iter()
+            .map(|k| Key::from_raw(k))
```

Comment:
Umm, this change introduces an additional allocation for `vec![]` but brings no benefit.

As it's in a test, it's okay to just make clippy happy. But it's curious to know what does clippy complain about the old code?

Reply:
Check [this](https://github.com/rust-lang/rust-clippy/issues/9101).

Reply:
Let's use `[b"c", b"a", b"b"].iter().copied()` just like other cases in this PR.